docs/hardening.md

```diff
@@ -0,0 +1,112 @@
+# Cluster Hardening
+
+If you want to improve the security on your cluster and make it compliant with the [CIS Benchmarks](https://learn.cisecurity.org/benchmarks), here you can find a configuration to harden your **Kubernetes** installation.
+
+To apply the hardening configuration, create a file (eg. `hardening.yaml`) and paste the content of the following code snippet into that.
+
+**N.B.** Some of these configurations have just been added to **Kubespray**, so ensure that you have the latest version to make it works properly.
+
+Also, ensure that other configurations doesn't override these.
+
+`hardening.yaml`:
+
+```yaml
+# Hardening
+---
+# This allow us to use PodSecurity admission plugin which is provided as built-in
+kube_version: v1.23.6
```

Comment:
I am not sure we need to specify `kube_version` here.
The above specified version is the same as the default value of Kubespray.
And we are updating it to a newer version always after releasing a new Kubernetes.

If having the `kube_version` value here, we need to update here also.
If there is not strong reason why we need to specify it here, it would be better to avoid specifying it.

Reply:
It was to set a kind of minimum requirement (as described in the comment PodSecurity admission plugin is available just from this version). If you think this could be misunderstood, I can remove it and write a note about minimum kubernetes verions.

Reply:
> I can remove it and write a note about minimum kubernetes verions.

Thanks for your explanation, the above seems a good idea for me.